buildSrc/linux.gradle

```diff
@@ -192,15 +200,18 @@ setupTools("linux_freetype_tools",
         def libs = properties.getProperty("libs")
         if (cflags && libs) {
             freetypeCCFlags.addAll(cflags.split(" "))
-            freetypeLinkFlags.addAll(libs.split(" "))
+            if (!IS_STATIC_BUILD) {
+                freetypeLinkFlags.addAll(libs.split(" "))
+            }
         } else {
             throw new IllegalStateException("Linux freetype packages not found.\nIf freetype pacakges are installed, please remove the build directory and try again.")
         }
     }
 )
 
 def compiler = IS_COMPILE_PARFAIT ? "parfait-gcc" : "${toolchainDir}gcc";
-def linker = IS_COMPILE_PARFAIT ? "parfait-g++" : "${toolchainDir}g++";
+def AR = "ar"
+def linker = IS_STATIC_BUILD ? AR : IS_COMPILE_PARFAIT ? "parfait-g++" : "${toolchainDir}g++";
```

Comment:
Do we need this extra def AR = "ar"? I think we can just use "ar" to be consistent with "parfait-g++".